### PR TITLE
Custom notificationCenter option added

### PR DESCRIPTION
**Pull request by Rabursky**
Pretty straightforward PR.

### CODE REVIEW

File: Reachability/Reachability.swift [hunk 1]
```diff
@@ -71,6 +71,8 @@ public class Reachability {
     public var whenReachable: NetworkReachable?
     public var whenUnreachable: NetworkUnreachable?
     public var reachableOnWWAN: Bool
+    
+    public var notificationCenter: NotificationCenter = NotificationCenter.default
```
Comment (ashleymills) [inline Reachability/Reachability.swift]: Hi - what's the reason for this change?

Comment (Rabursky) [inline Reachability/Reachability.swift]: Personally I am using non-default notification centre as a bus for domain-specific events within my app. I believe it is just easier to track those events in the separate one.
Those are just 2 lines and still gives some additional flexibility.

Comment (ashleymills) [inline Reachability/Reachability.swift]: OK, thanks. So future updaters know why this has been done, please add a comment to the property to explain and I'll merge

Comment (Rabursky) [inline Reachability/Reachability.swift]: Comment added. I hope it is explanatory enough. Thanks!

**Comment by ashleymills** [comment]
@Rabursky Thanks for the PR 

Cheers
Ash
